projects/demo/src/modules/app/app.routes.ts

```diff
@@ -867,14 +849,6 @@ export const ROUTES: Routes = [
             title: `Preview`,
         },
     },
-    {
-        path: `components/svg`,
-        loadChildren: async () =>
-            (await import(`../components/svg/svg.module`)).ExampleTuiSvgModule,
-        data: {
-            title: `Svg`,
```

Comment:
Where is there previous page `Svg` ?

https://taiga-ui.dev/components/svg

Reply:
Is it this one ?
https://taiga-ui--pr4488-splincode-feat-icons-mfcciomp.web.app/icons/bundled

Reply:
API in https://taiga-ui--pr4488-splincode-feat-icons-mfcciomp.web.app/icons/API/API
Examples in https://taiga-ui--pr4488-splincode-feat-icons-mfcciomp.web.app/icons/bundled